assets/kube-scheduler/kube-scheduler-deployment.yaml

```diff
@@ -55,6 +55,9 @@ spec:
         - "--authentication-kubeconfig=/etc/kubernetes/secret/kubeconfig"
         - "--authorization-kubeconfig=/etc/kubernetes/secret/kubeconfig"
         - "-v=2"
+{{if .ExtraFeatureGates }}{{ range $featureGate := .ExtraFeatureGates }}
+        - "--feature-gates={{ $featureGate }}"
```

Comment:
should we not statically include feature gates that are not in .ExtraFeatureGates?
https://github.com/openshift/hypershift-toolkit/blob/master/assets/kube-apiserver/config.yaml#L28-L30

Reply:
Even better, specify them all in the config and just use the ones from the config everywhere, that way we make sure they're consistent across components

Reply:
done